assets/helpers/utilities.js

```diff
@@ -87,6 +87,10 @@ function validateEmailAddress(email: string): boolean {
   return emailValidationRegex.test(email);
 }
 
+function validInputField(input: ?string): boolean {
```

Comment:
I would rename this function to be `inputFieldNotEmpty` or something else which implies that we are checking that the input field is not empty, since `validInputField` name does not specify what consider to be a valid field.

Reply:
Good point @svillafe - renaming this made me realise that it's easier to follow if the logic is flipped (otherwise we end up writing !inputFieldNotEmpty), so I've changed that too.